packages/foreman/rubygem-actionpack/rubygem-actionpack.spec

```diff
@@ -17,7 +17,7 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 2.2.2
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(activesupport) = 5.2.1
+Requires: %{?scl_prefix}rubygem(activesupport) = 6.0.2.1
```

Comment:
Why wasn't this done as part of the bumping? `bump_rpm.sh` should have taken care of this by rewriting the dependencies. Was the automation not used or didn't it work?

Reply:
I don't think it worked, I used the automation for it, this was done back in January.

Reply:
So, I used obal update instead of bump_rpm, I didn't realize I should have used it instead.

Reply:
Yes, obal update is IMHO unsupported because it lacks these crucial features. Given there's a new patch version out I'd suggest to start a fresh branch and bump all the dependencies using `bump_rpm.sh`.